xml/System/Random.xml

```diff
@@ -722,7 +724,7 @@ The following example uses the parameterless constructor to instantiate three <x
  The following example demonstrates how to use the <xref:System.Random.NextBytes%2A> method to fill an array of bytes with random byte values.  
```

Comment:
Same here but in this case you'll also have to create the MemberGroup element for NextBytes since it doesn't exist in the PR right now

Reply:
Done.